SU-HSDO/suhumsci
# ARCH-265: Use the block description instead of the admin label

# READY FOR REVIEW

# Summary
- Allow site managers the ability to edit block content entities
- Use the block "Description" instead of the admin label on the display.

# Needed By (Date)
- End of sprint sept 27

# Urgency
- Low

# Steps to Test
1. checkout this branch
1. `composer update --prefer-source`
1. sync with partial `blt drupal:sync --site=archaeology --partial`
1. either use masquarade (form on the top of all admin pages) or drush to view the site as `Howard`
1. On the homepage, ensure when you hover over the blocks in the footer, that the contextual links allow you to edit the block
1. change the block description
1. verify the edited description displays on the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
